<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordInput</class>
 <widget class="QDialog" name="PasswordInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>383</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>383</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyPassWordLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确 认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_CLR">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>CLR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_0">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_BS">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>81</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退格</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>BS</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPassWordLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">dialog/mypasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
